backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -127,9 +127,13 @@ export class TransactionResolver {
       transactions.push(new Transaction(userTransaction, self, linkedUser))
     })
 
+    const toHoldAvailable = await holdAvailable(user.id, now)
+
     // Construct Result
     return new TransactionList(
-      calculateDecay(lastTransaction.balance, lastTransaction.balanceDate, now).balance,
+      calculateDecay(lastTransaction.balance, lastTransaction.balanceDate, now).balance.minus(
```

Comment:
What about the balance resolver?

Reply:
I know that this is not the best solution. Can I improve this in following PRs?